.github/workflows/monodevelop.yml

```diff
@@ -9,7 +9,7 @@ on:
       - '*'
 
 jobs:
-  build:
+  linux:
```

Comment:
Nice. Leave room for macos and windows in the future if anyone can contribute on those platforms.

Reply:
the jobs for windows and macos can be named 'windows' and 'macos'